File: src/model/Network.kt
package moe.msm.model

import moe.msm.ValidationException
import java.util.*

/**
 * @author sam_nya (privateamusement@protonmail.com)
 */
class Network(
    val id: Long?,
    val uuid: UUID?,
    val name: String?,
    val machine: Machine?
) {
    fun validate(): Boolean = if (name != null) true else throw ValidationException("Name is required")
}

File: Linkmon-desktop/model/Network.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Linkmon_desktop.model
{
    public class Network
    {
        public long Id { get; set; }
        public string Uuid { get; set; }
        public string Name { get; set; }
        public Machine Machine { get; set; }
    }
}
